support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListData.tsx

```diff
@@ -32,10 +32,12 @@ export const getSvgIcon = (isUnlocked: boolean): JSX.Element =>
 		<SvgCrossRound isAnnouncedByScreenReader size="small" />
 	);
 
-export const checkListData = ({ higherTier }: TierUnlocks): CheckListData[] => {
+export const checkListData = (
+	{ higherTier }: TierUnlocks,
+	isemotionalBenefitTestVariant: boolean,
+): CheckListData[] => {
 	const maybeGreyedOutHigherTier = higherTier ? undefined : greyedOut;
-
-	return [
+	const dataWithoutEmotionalBenefits = [
```

Comment:
Instead of the using the intermediate `dataWithoutEmotionalBenefits` var, you could conditionally spread the object into the original array that was being being returned here e.g. 
```suggestion
	return [
      ...(isemotionalBenefitTestVariant ? [
				{
					icon: getSvgIcon(true),
					text: (
						<p>
							<span css={boldText}>Make a bigger impact. </span>Sustain open,
							independent journalism long term
						</p>
					),
				},
		  ]
		: []),
```

Reply:
Have updated as per suggestion